apps/camera/js/lib/camera/camera.js

```diff
@@ -846,6 +846,10 @@ Camera.prototype.takePicture = function(options) {
 };
 
 Camera.prototype.updateFocusArea = function(rect, done) {
+  if (!this.mozCamera) {
+    debug('will focus but this.mozCamera is null, return to ignore focus');
```

Comment:
This log is confusing to me. I think the code is self explainatory. Please change to:

```js
if (!this.mozCamera) { return; }
this.focus.updateFocusArea(rect, focusDone);
...
```